doc/sphinx/start_install.rst

```diff
@@ -197,15 +197,17 @@ Framework interaction with Conda environments
 
 As just described in :ref:`ref-framework-sample`, when you run the ``mdtf`` executable, among other things, it reads ``pod_list`` in ``default_tests.jsonc`` and executes POD codes accordingly. For a POD included in the list (referred to as $POD_NAME):
 
-1. The framework will first try to determine whether there is a Conda environment named ``_MDTF_$POD_NAME`` under ``$CONDA_ENV_DIR``. If yes, the framework will switch to this environment and run the POD.
+1. The framework will first try to look for the YAML file ``src/conda/env_$POD_NAME.yml``. If it exists, the framework will assume that the corresponding Conda environment ``_MDTF_$POD_NAME`` has been installed under ``$CONDA_ENV_DIR``, and will switch to this environment and run the POD.
 
-2. If not, the framework will then look into the POD's ``settings.jsonc`` file in ``$CODE_ROOT/diagnostics/$POD_NAME``. ``runtime_requirements`` in ``settings.jsonc`` specifies the programming language(s) adopted by the POD:
+2. If not, the framework will then look into the POD's ``settings.jsonc`` file in ``$CODE_ROOT/diagnostics/$POD_NAME/``. The ``runtime_requirements`` section in ``settings.jsonc`` specifies the programming language(s) adopted by the POD:
 
-    a). If purely Python, the framework will switch to ``_MDTF_python3_base`` and run the POD (`_MDTF_python2_base` for ealier PODs developed in Python 2.7).
+    a). If purely Python 3, the framework will look for ``src/conda/env_python3_base.yml`` and check its content to determine whether the POD's requirements are met, and then switch to ``_MDTF_python3_base`` and run the POD.
```

Comment:
If purely Python 3, the framework will look for ``src/conda/env_python3_base.yml``<s> and </s>,  check its content to determine whether the POD's requirements are met, <s>and</s> then switch to ``_MDTF_python3_base`` and run the POD.